### PR TITLE
React/Proptypes: Add additional video resource

### PR DESCRIPTION
**Pull request by scottwright-dev**
## Because

This PR adds an additional video resource which provides a brief overview of PropTypes and some various use cases  of how to use PropTypes in React.

## This PR

- Add a link to the video resource in the Additional Resources section.

## Issue

Closes: n/a

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: react/the_react_ecosystem/type_checking_with_proptypes.md [hunk 1]
```diff
@@ -73,37 +73,37 @@ RenderName.defaultProps = {
...
 - <a class="knowledge-check-link" href="#using-defaultprops">If we pass in a prop to a component that has a defaultProp defined, what would happen?</a>
 - <a class="knowledge-check-link" href="https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application">What is the difference between PropTypes and TypeScript?</a>
 
 ### Additional resources
 
-This section contains helpful links to related content. It isn’t required, so consider it supplemental.
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- If you find video lessons easier to absorb, consider watching [Web Dev Simplified's video on understanding PropTypes in React](https://www.youtube.com/watch?v=cx0S8JyiVxc). This video covers many of the topics discussed in this lesson.
```
Comment (thatblindgeye) [inline react/the_react_ecosystem/type_checking_with_proptypes.md:109]: Just wondering, does this video go into additional topics that this lesson doesn't cover, or is it just a video alternative to what this lesson presents?

File: react/the_react_ecosystem/type_checking_with_proptypes.md [hunk 2]
```diff
@@ -73,37 +73,37 @@ RenderName.defaultProps = {
...
-1. You can even set up custom validators in PropTypes. Read [this article on using PropTypes on LogRocket](https://blog.logrocket.com/validate-react-props-proptypes/) for a more in-depth look into the benefits and use cases of PropTypes.
-1. [This StackOverflow thread goes into the differences of TypeScript and PropTypes. ](https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application)Give it a read!
+1. You can even set up custom validators in PropTypes. For a more in-depth look into the benefits and use cases of PropTypes, read this comprehensive guide [Validating React Props with PropTypes on LogRocket](https://blog.logrocket.com/validate-react-props-proptypes/).
+1. Dive into the nuanced comparison between PropTypes and TypeScript for type safety in React with [a detailed analysis found on StackOverflow](https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application), highlighting key differences and use cases.
```
Comment (thatblindgeye) [inline react/the_react_ecosystem/type_checking_with_proptypes.md:93]:
```suggestion
1. Dive into the nuanced comparison between PropTypes and TypeScript for type safety in React with this [StackOverflow post on differences between PropTypes and TypeScript](https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application).
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

**Comment by thatblindgeye** [comment]
@scottwright-dev just checking in to see if you've seen the comments above?

File: react/the_react_ecosystem/type_checking_with_proptypes.md (hunk 2 above)
Comment (scottwright-dev) [reply react/the_react_ecosystem/type_checking_with_proptypes.md:93]: Hi @thatblindgeye, apologies for the delay, sure thing I will add these suggestions.

**Comment by scottwright-dev** [comment]
> @scottwright-dev just checking in to see if you've seen the comments above?

Hi @thatblindgeye, apologies for the delay. in responding. I've just incorporated your suggestions into the commit. Please let me know if I need to do anything else.

Many Thanks

File: react/the_react_ecosystem/type_checking_with_proptypes.md (hunk 1 above)
Comment (thatblindgeye) [reply react/the_react_ecosystem/type_checking_with_proptypes.md:109]: @scottwright-dev depending on the answer to the above, I'm inclined to not include this video if it's just going over the same content in the lesson, just as a video. I feel like that's something a user could seek out on their own if they want a video.

Comment (scottwright-dev) [reply react/the_react_ecosystem/type_checking_with_proptypes.md:109]: HI @thatblindgeye, it is essentially a summary of whats discussed in the lesson so feel free to omit.

Comment (thatblindgeye) [reply react/the_react_ecosystem/type_checking_with_proptypes.md:109]: Gotcha. I'd say let's revert this link back to the stock verbiage then.

Comment (scottwright-dev) [reply react/the_react_ecosystem/type_checking_with_proptypes.md]: @thatblindgeye sure thing, all done!

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**